api/librdb-api.h

```diff
@@ -262,6 +265,8 @@ typedef struct RdbHandlersStructCallbacks {
 
     /*** TODO: RdbHandlersStructCallbacks: ***/
 
+    /* Callback to handle an item from a plain sorted set */
+    RdbRes (*handleZsetPlain)(RdbParser *p, void *userData, RdbBulk item, double score);
     /* Callback to handle a ziplist-based sorted set value */
     RdbRes (*handleZsetZL)(RdbParser *p, void *userData, RdbBulk ziplist);
     /* Callback to handle a listpack-based sorted set value */
```

Comment:
Please move all the supported zset callbacks above the line : `/*** TODO: RdbHandlersStructCallbacks: ***/`

Reply:
done